### ISSUE TITLE
User management

### ISSUE THREAD

**Issue by brytannia** [brytannia/sloboda-edu-dashboard#5]
If user doesn't registered yet, during creation event admin can create one. When user registered, Admin can merge accounts.

**Comment by alvarofernandoms** [comment]
This merge will be manually, via ActiveAdmin interface?

**Comment by brytannia** [comment]
@alvarofernandoms yes. Admin choose two account and merge them.In case of conflicts, information is taken from that profile which was created by user himself.
